### ISSUE TITLE
KDE: unintuitive dependency on konqueror

### ISSUE THREAD

**Issue by lexelby** [copiousfreetime/launchy#102]
I'm running Kubuntu 15.04, which includes KDE 4.14.6.  Launchy correctly detects that the session is KDE, and it tries to run `kfmclient <url>` when asked to launch a browser.  I've set my default browser in KDE, gnome, and any other place I can think of to chromium.  Apparently I don't even have konqueror installed, and apparently the kfmclient utility is part of konqueror.

The result is that launchy falls back to its static list of browsers and opens URLs in firefox.  Installing konqueror fixes the problem.  It's a bit weird to have to install a browser that isn't my preferred one in order to get my preferred one detected properly.

`xdg-open` works fine on my system and opens URLs in chromium.  Perhaps fall back to `xdg-open` if `kfmclient` doesn't exist?

**Comment by copiousfreetime** [comment]
@lexelby Interesting, I personally don't have a KDE machine to do this testing on. According to the documentation (https://techbase.kde.org/Development/Tools/Using_kfmclient):

> kfmclient is a tool for opening URL's from the command line, or from scripts. kfmclient is part of the kdebase package, so you can rely on it's presence on any system running KDE. 

I ponder if this indicates a bug in the packaging of KDE on Kubuntu. 

In any case, I think we can accommodate a fallback to `xdg-open` should `kfmclient` not be found. I'll push up a fix, and if you can test it out I would be grateful.

**Comment by copiousfreetime** [comment]
@lexelby please test this fix. Probably easiest to just monkey patch what you have already, its a 1 line change. See 97eecfd

**Comment by lexelby** [comment]
I'm sorry, but I really can't seem to test this :/  I made the change and removed konqueror, but launchy continued to use Firefox.  I added a debug logging statement and set the LAUNCHY_DEBUG environment variable, and my statement doesn't appear in the output, so I think it's fairly clear that I haven't found the actual copy of the launchy gem that is being used.

I instead tested by unsetting KDE_FULL_SESSION, and launchy is able to properly load URLs in chrome, so it must be able to fall back to xdg properly.  Seems good enough to me.
